multimodal/vl2l/src/mlperf_inference_multimodal_vl2l/task.py

```diff
@@ -16,8 +16,19 @@
 from datasets import load_dataset
 from loguru import logger
 from openai import AsyncOpenAI, DefaultAioHttpClient
+from pydantic import BaseModel
 from pympler import asizeof
 
+
+class ProductMetadata(BaseModel):
```

Comment:
Maybe this is a good time to move all the `BaseModel` into a separate `type.py` or something. It would also solve your circular import problem earlier.